### ISSUE TITLE
CRAN: NOTEs on the "HTML version of manual" check

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/globals#84]
---------- Forwarded message ---------
From: Kurt Hornik
Date: Mon, Nov 7, 2022 at 5:41 AM
Subject: CRAN package globals
Cc: <CRAN@r-project.org>

Dear maintainer,

Please see the problems shown on
<https://cran.r-project.org/web/checks/check_results_globals.html>.

Specifically, see the NOTEs for the "HTML version of manual" check.

These are from a recent change to r-devel which now uncovers duplicated
\item entries in \arguments as part of the HTML refman validation check.

Please fix by removing the duplicated \item entries.

Please correct before 2022-11-21 to safely retain your package on CRAN.

Best,
-k

**Comment by HenrikBengtsson** [comment]
Details:

```
Version: 0.16.1
Check: HTML version of manual
Result: NOTE
    Found the following HTML validation problems:
    walkAST.html:53:9 (walkAST.Rd:23): Warning: <code> anchor "name" already defined
    walkAST.html:57:9 (walkAST.Rd:25): Warning: <code> anchor "call" already defined
    walkAST.html:61:9 (walkAST.Rd:27): Warning: <code> anchor "pairlist" already defined
```

**Comment by HenrikBengtsson** [comment]
Fixed in commit 62d4191.
